src/rgw/rgw_op.h

```diff
@@ -1065,7 +1065,7 @@ class RGWGetACLs : public RGWOp {
 
 class RGWPutACLs : public RGWOp {
 protected:
-  size_t len;
+  int len;
```

Comment:
so I see that though rgw frontends do return a `size_t` when reading input via `recv_body`, the conversion seems to happen at `RGWRestfulIO which either returns the size_t converted as int or the error code, while we expect most of recv_body to fit in an int, maybe a better way out would be to accept the received size as a size_t itself, @cbodley @yehudasa thoughts?